R/geographicmap.R

```diff
@@ -475,12 +489,15 @@ leafletMap <- function(coords, colors, opacity, min.value, max.range, color.NA,
         map <- setView(map, -96, 37.8, zoom = 4)
     } else if ("longitude" %in% colnames(coords@data)) {
 
-        # Manually set zoom level if we cross the longitude = 180 line
+        # Manually set zoom if map we are close to the antimeridian
```

Comment:
There seems to be a typo in the comment?

Reply:
Fixed now, thanks